pkg/controller/provider/container/openstack/watch.go

```diff
@@ -302,7 +304,10 @@ func (r *VMEventHandler) validated(batch []*policy.Task) {
 		if task.Error != nil {
 			r.log.Error(
 				task.Error, "VM validation failed.")
-			continue
+
+			if len(task.Concerns) == 0 {
```

Comment:
Please add a comment that if there are concerns we need to update and commit the changes